form.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Registration Form</title>
+</head>
+<body>
+
+<form action="/submit-form" method="post">
+
+    <fieldset>
+  <legend>Basic User Information:</legend>
+  <label for="fullname">Full Name:</label>
+  <input type="text" id="fullname" name="fullname" required autofocus>
+  <br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="email" required pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}">
+  <br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="password" required>
+  <label for="confirm-password">Confirm Password:</label>
```

Comment:
please always align the tags. All child tags should be moved 1 tab to right.

Reply:
Well noted. Thank you.